### PR TITLE
Fix #55: Resolved ads do not adhere to daily_cap, per_day or total_max

### LINKED ISSUE #55: Ads do not adhere to daily_cap, per_day or total_max

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

File: include/bat/ads/ad_info.h [hunk 1]
```diff
@@ -20,8 +20,12 @@ struct ADS_EXPORT AdInfo {
   bool FromJson(const std::string& json);
 
   std::string creative_set_id;
+  std::string campaign_id;
   std::string start_timestamp;
   std::string end_timestamp;
+  uint64_t daily_cap;
```
Comment (bridiver) [inline include/bat/ads/ad_info.h]: I thought these weren't going to be 64?

Comment (bridiver) [reply include/bat/ads/ad_info.h]: just unsigned int?

Comment (tmancey) [reply include/bat/ads/ad_info.h]: 👍

**Comment by tmancey** [comment]
Fixes https://github.com/brave-intl/bat-native-ads/issues/55

**Pull request merged**